### ISSUE TITLE
[Bug] realy outdated package

### ISSUE THREAD

**Issue by mrmedia** [creativetimofficial/vue-soft-ui-dashboard-laravel#1]
### Version
1.0.0
### Reproduction link
https://localhost
### Operating System
Mac OS Ventura 13.4.1
### Device
Mac OS
### Browser & Version
Chrome Latest
### Steps to reproduce
- Unzip package
- Try to set it up
### What is expected?
- A barely deprecated package for the price
### What is actually happening?
I'v joined / bought the Creative Tims Club Membership and downloaded Vue Soft UI Dashboard Laravel product from your website.

I were a little shocked because the implementation is really not up to date. Your website says "Release one month ago"

- The package for example uses bable-eslint insted of bable/eslint-parser which is deprecated
- The vue multiword rule which was introduced with vuejs/eslint-plugin-vue v7.20.0 ..... 20 months ago is not etablished in the vue files of the downloaded package.

The information of the release dates on your website does not correspond to the facts, which is very annoying for the money.

---
### Solution
Manually update each downloaded product

**Comment by ad84** [comment]
@mrmedia Agreed. I'm glad I saw this and tired to build this free repo before paying any money. 
This repo is out of date, and it seems by your comments the "pro" paid version is no better. 

NPM or yarn will not install the packages required for the front end using node 18 the current LTS. End of life for node 16 is very soon - https://nodejs.org/en/blog/announcements/nodejs16-eol 
If this is the newest Vue based free template on the website it doesn't bode well for their other products.
